eGovPDX/portlandor
# POWR-1555 commenting out unused bootstrap components

Disabling imports for unused bootstrap components in the style.scss file.

These are the dependancies removed-

- Root
- Code
- Dropdown
- Button-group
- Custom-forms
- Jumbotron
- Progress
- Toasts
- Modal
- Popover
- Carousel
- Spinners

To Test-
- Go to https://powr-1555-portlandor.pantheonsite.io/.
- Check the homepage, services main page, bureaus and offices main page, and alerts main page at different breakpoints to ensure there is no style regression.
